matomo.module

```diff
@@ -468,9 +468,9 @@ function matomo_user_presave($account) {
 function matomo_cron() {
   $config = config('matomo.settings');
   // Regenerate the matomo.js every day.
-  if (REQUEST_TIME - $config->get('last_cache') >= 86400 && $config->get('cache')) {
+  if (REQUEST_TIME - state_get('matomo_last_cache', 0) >= 86400 && $config->get('cache')) {
     _matomo_cache($config->get('url_http') . 'matomo.js', TRUE);
-    $config->set('last_cache', REQUEST_TIME);
+    state_set('matomo_last_cache', REQUEST_TIME);
     $config->save();
```

Comment:
As it's a state now, this `$config->save();` is obsolete now. We don't touch config anymore in cron runs.

Reply:
Good point. Updated.